__tests__/app.test.js

```diff
@@ -96,6 +90,28 @@ describe('/api/articles', () => {
         });
       });
   });
+  test('TASK 11 GET:200 and specific article', () => {
+    return request(app)
+      .get('/api/articles?topic=cats')
+      .expect(200)
+      .then(({ body }) => {
+        expect(body[0].topic).toBe('cats');
+      });
+  });
+  test('TASK 11 GET:200 and all articles with topic mitch', () => {
+    return request(app)
+      .get('/api/articles?topic=mitch')
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.length).toBe(12);
+        body.forEach((article) => {
+          expect(article.topic).toBe('mitch');
+        });
+      });
+  });
+  test('TASK 11 GET:204 on /api/articles? when query adresses to non existing row', () => {
+    return request(app).get('/api/articles?topic=nonsens').expect(204);
+  });
```

Comment:
Refactor this to be a 404 - the topic 'nonsens' doesn't exist yet, so it cannot be found :) 

Also, there is a 'paper' topic which exists but has no articles, you should build a 200 test for that, which should respond with an empty array :)